grammar.js

```diff
@@ -57,7 +57,8 @@ module.exports = grammar({
     $._keyword_colon,
     $._unary_minus,
     $._binary_minus,
-    $._binary_star
+    $._binary_star,
+    $._singleton_class_left_angle_left_langle
```

Comment:
If you want, you could just put the string `'<<'` here in the externals list, rather than introducing the name `_singleton_class_left_angle_left_langle`.

Reply:
Nope, nevermind. As @tclem explained in chat, there appear to be special rules just for `<<` after the word, `class`.